docs/en/observability/profiling-self-managed-ops.asciidoc

```diff
@@ -262,3 +263,85 @@ Download the corresponding binary version and replace the existing one, using th
 Replace the old binary and restart the services.
 
 You will find the links to the new binaries in the "Add Data" page, under the "Binary" tab.
+
+[discrete]
+[[profiling-self-managed-kubernetes-tips]]
+== Kubernetes tips
+
+When deploying the Universal Profiling backend on Kubernetes, there are some best practices to follow.
+
+[discrete]
+=== Ingress configuration
+
+If you are using an ingress controller, the connection routing to the collector Service should be configured to use the gRPC protocol.
+
+We provide an `Ingress` resource as part of the Helm chart, and because the ingress can be any implementation,
+you should provide to the chart values the configuration of the ingress, adding the class name and the
+necessary annotations using the `ingress.annotations` field.
```

Comment:
```suggestion
We provide an `Ingress` resource as part of the Helm chart. Because the ingress can be any implementation,
you must configure the controller with a class name and
any necessary annotations using the `ingress.annotations` field.
```